src/switch_core_media.c

```diff
@@ -12248,6 +12252,10 @@ SWITCH_DECLARE(void) switch_core_media_gen_local_sdp(switch_core_session_t *sess
 			}
 		}
 
+		if (switch_channel_var_true(session->channel, "use_rtcp_xr") || (!switch_channel_var_false(session->channel, "use_rtcp_xr") && smh->mparams->offer_rtcp_xr)) {
```

Comment:
Should this be inside the validation of `rtcp_audio_interval_msec`? If we disable rtcp then we shouldn't send or offer rtcp-xr.

Reply:
ok, makes sense